### PR TITLE
deno2: build on Windows

### PR DESCRIPTION
**Pull request by piscisaureus**

### CODE REVIEW

**Comment by jedahan** [comment]
Is there a way to check if the target is a directory without having to pass that flag in?

**Comment by piscisaureus** [comment]
@jedahan, no, because the target may not exist when the symlink is created.

File: deno2/.gclient [hunk 1]
```diff
@@ -11,7 +11,6 @@ solutions = [{
         'v8/testing/gmock': None,
         'v8/tools/swarming_client': None,
         'v8/third_party/instrumented_libraries': None,
-        'v8/tools/gyp': None,
```
[the remark this answers is not in the document]
Comment (piscisaureus) [reply deno2/.gclient]: v8 has a dependency on it through the automatic clang downloader.
See `v8/tools/clang/scripts/update.py`.

File: deno2/BUILD.gn [hunk 2]
```diff
@@ -112,7 +112,7 @@ run_node("run_tsc") {
     ":pbjs_hack",
   ]
   args = [
-    "./node_modules/.bin/tsc",
+    "./node_modules/typescript/bin/tsc",
```
[the remark this answers is not in the document]
Comment (piscisaureus) [reply deno2/BUILD.gn]: Symlinks are not the issue here, but the issue is that npm (and also yarn) generate some bash (!) script for ./bin/tsc" (not a javascript file with shebang as they do on other OS).

**Pull request merged**